SECURITY.md

```diff
@@ -6,22 +6,22 @@ issues.
 
 This policy is adapted from the policies of the following CNCF projects:
 
-* [Envoy](https://github.com/envoyproxy/envoy)
-* [CoreDNS](https://github.com/coredns/coredns)
+- [Envoy](https://github.com/envoyproxy/envoy)
```

Comment:
Why change all the bullets from `*` to `-`? In the past we had some tooling that converted it to `*`, so i'm reluctant to see it changed again to `-`.

Reply:
It's just Prettier editing it on save actually

Reply:
I am going into the file documentation now

Reply:
@galexrt Just converted these a while back, so we need to make sure we're using consistent tooling so we don't keep changing them. If I change it to a `-` in vscode, I see the linter complain, so we wouldn't want these changes unless we change our tools:
```
MD004/ul-style: Unordered list style [Expected: asterisk; Actual: dash]markdownlint[MD004](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md004)
```

Reply:
@travisn I think there is an option in the `markdownlint` of VSCode to change the unordered list style, though I have to say we should commit to one as changing that option will cause these linter complaints in the unchanged documents.

I would prefer to stay with `*` and look to changing the `markdownlint` config for "everyone" working on Rook if possible.